frontend/packages/knative-plugin/src/models.ts

```diff
@@ -3,6 +3,7 @@ import {
   chart_color_red_300 as knativeEventingColor,
 } from '@patternfly/react-tokens';
 import { K8sKind } from '@console/internal/module/k8s';
+import { BadgeType } from '@console/shared/src/components/badges/badge-factory';
```

Comment:
```suggestion
import { BadgeType } from '@console/shared';
```

Reply:
getting error while using `'@console/shared'`
```
Uncaught ReferenceError: Cannot access 'allModels' before initialization
    at Module.allModels (k8s-models.ts:1)
    at Module../public/actions/ui.ts (ui.ts:99)
    at __webpack_require__ (bootstrap:795)
    at fn (bootstrap:150)
    at Module../public/reducers/ui.ts (ui.ts:1)
    at __webpack_require__ (bootstrap:795)
    at fn (bootstrap:150)
    at Module../public/components/utils/kebab.tsx (k8s-watcher.js:38)
    at __webpack_require__ (bootstrap:795)
    at fn (bootstrap:150)
```

Reply:
yeah will stick with full imports, it may happen as this import is happening in models which is called in plugin for registration so with higher level import it seems trying to invoke `allModels` before initialization